build.gradle

```diff
@@ -52,6 +52,11 @@ def createSpotlessTarget = { pattern ->
         'dbt-project-template',
         'tools'
     ]
+
+    if(System.getenv().containsKey("CORE_ONLY")) {
```

Comment:
This is required because we'll have file differences if we apply `spotless` but not `black`